lab/docker/importer/volume/properties/datasource.password

```diff
@@ -0,0 +1 @@
+clinlims
```

Comment:
@mozzy11 just pointing out another location for hard-coded passwords, which might get messy